src/app.js

```diff
@@ -1 +1,34 @@
+/* eslint-disable no-console */
 'use strict';
+
+const readline = require('readline');
+const { bullAndCows } = require('./bull_and_cows');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
```

Comment:
Move this to a separate module for input and output operation, as stated in requirements.